### DISCUSSION TITLE
Octodash sleeping

### DISCUSSION THREAD

**Discussion by PierreGdrlt** [UnchartedBull/OctoDash#1707] [Q&A]
How can I access the Octodash menu even if the printer is out of service?  I created an icon that allows me to put the printer ON, but Octodash refuses me access because the printer is OFF.

**Comment by UnchartedBull** [comment]
V3.0.0 will support this. The beta (which has this feature included) is available on the Discord Server in announcements :)

**Comment by PierreGdrlt** [comment]
Thank you.  I'm waiting for the new release !!!
